.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: python
+dist: xenial
+sudo: true
+python:
+  - "3.6"
```

Comment:
Won't you support Python 2? 😱

Reply:
Because [RIP](https://pythonclock.org/).

Reply:
![](https://i.pinimg.com/originals/57/0c/ca/570ccadc6f792041a972d425703bffb8.jpg)